### DISCUSSION TITLE
How to assign a static IP to ESP Thread Border Router

### DISCUSSION THREAD

**Discussion by mforbiz** [openthread/openthread#10738] [General]
Hi.
I've succesfully installed ESP Thread Border Router on ESP32-H2.
Alll works fine and the router receive a dynamic IP from local DHCP server and I can access it via web GUI.

![image](https://github.com/user-attachments/assets/700cfc9e-fdb8-4a5f-9431-a14f8e090d32)

Anyway I'd like to assign to the router a static , local IP Address.

Is it possible ?
And, if yes, how ?
Any idea?
Thanks in advance.

**Comment by chshu** [comment]
The ESP32-S3 on the ESP Thread BR is a Wi-Fi station, its IP allocation is managed by the Wi-Fi AP. You can modify your Wi-Fi AP settings to use a static IP instead of DHCP.
